site/en/install/lang_java.md

```diff
@@ -1,207 +1,203 @@
+  <version>0.2.0</version>
 </dependency>
 ```
 
-### GPU support
+You can leave the `extension` variable empty to use a vanilla version of TensorFlow or set it to one
+of the supported variant:
+* `-mkl`: Support for Intel® MKL-DNN on all platforms
+* `-gpu`: Support for CUDA® on Linux and Windows platforms
+* `-mkl-gpu`: Support for Intel® MKL-DNN and CUDA® on Linux and Windows platforms.
 
-If your system has [GPU support](./gpu.md), add the following TensorFlow
-dependencies to the project's `pom.xml` file:
+Optionally, you can also add a dependency to the `tensorflow-framework` library, which provides a rich
+set of high-level utilities to improve the developer experience with with TensorFlow-based machine learning
+on the JVM.
 
 ```xml
 <dependency>
   <groupId>org.tensorflow</groupId>
-  <artifactId>libtensorflow</artifactId>
-  <version>2.3.0</version>
-</dependency>
-<dependency>
-  <groupId>org.tensorflow</groupId>
-  <artifactId>libtensorflow_jni_gpu</artifactId>
-  <version>2.3.0</version>
+  <artifactId>tensorflow-framework</artifactId>
+  <version>0.2.0</version>
 </dependency>
 ```
 
-### Example program
+### Reducing Number of Dependencies
+
+It is important to note that adding a dependency to a `tensorflow-core-platform` artifact will import native 
+libraries for all supported platforms, which can increase significantly the size of your project.
+
+When it has already been established that your project will only run on a subset of these platforms, it is possible
+to exclude the artifacts of the other platforms using the 
+[Maven Dependency Exclusion](https://maven.apache.org/guides/introduction/introduction-to-optional-and-excludes-dependencies.html#dependency-exclusions) 
+feature.
+
+Another way to select which platforms you want to include in your application is to set JavaCPP system properties,
+in your Maven command line or in your `pom.xml`. Please see JavaCPP 
+[documentation](https://github.com/bytedeco/javacpp-presets/wiki/Reducing-the-Number-of-Dependencies) for more details.
+
+### Using Snapshots
+
+TensorFlow Java snapshots that reflect that latest version of the source code
+in [TensorFlow Java Repository](https://github.com/tensorflow/java) are available on [OSS Sonatype](https://oss.sonatype.org). 
+To depend on these artifacts, make sure to configure the OSS snapshots repository in your `pom.xml`.
+
+```xml
+<repositories>
+    <repository>
+        <id>tensorflow-snapshots</id>
+        <url>https://oss.sonatype.org/content/repositories/snapshots/</url>
+        <snapshots>
+            <enabled>true</enabled>
+        </snapshots>
+    </repository>
+</repositories>
+
+<dependencies>
+    <dependency>
+        <groupId>org.tensorflow</groupId>
+        <artifactId>tensorflow-core-platform</artifactId>
+        <version>0.3.0-SNAPSHOT</version>
+    </dependency>
+</dependencies>
+```
+
+## Installing with Gradle
+
+Same dependencies as with Maven can be added to your Gradle project to run TensorFlow. 
```

Comment:
"To use TensorFlow Java with Gradle you can include the following snippet in your `build.gradle`."